boa3_test/examples/README.md

```diff
@@ -0,0 +1,44 @@
+# Examples
+
+This folder has some examples of smart contracts that are used throughout the blockchain. It's also recommended checking
+the tests on boa3_test/tests/examples_tests, since they have comments describing why the methods are being called, and 
```

Comment:
```suggestion
the [tests](../tests/examples_tests), since they have comments describing why the methods are being called, and 
```
I think that linking to the folder is better for the user navigation through the files